### ISSUE TITLE
The link template options cannot be overwritten

### ISSUE THREAD

**Issue by ghost** [kartik-v/yii2-widget-sidenav#3]
Hi, i am trying to add data-pjax="0" to the link, could you fix this error?
 Thank you

**Comment by ilgiz-badamshin** [comment]
Also need possibility to override parent item template, it is hardcoded: 
`'<a href="{url}" class="kv-toggle">{icon}{label}</a>'`

**Comment by Nayank9** [comment]
Navigation side bar not showing correctly.

`<?php
use yii\helpers\Url;
use kartik\sidenav\SideNav;
use yii\helpers\Html;
use yii\widgets\Breadcrumbs;
use frontend\assets\AppAsset;
use common\widgets\Alert;
AppAsset::register($this);
?>
<?php $this->beginPage() ?>
<!DOCTYPE html>
<html lang="<?= Yii::$app->language ?>">
<head>
    <meta charset="<?= Yii::$app->charset ?>">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <?= Html::csrfMetaTags() ?>
    <title><?= Html::encode($this->title) ?></title>
    <?php $this->head() ?>
</head>
<body>
<?php $this->beginBody() ?>
  <?php
    $menuItems = [
        /_['label' => 'Home', 'url' => ['/site/index']],_/
        /_['label' => 'Contact', 'url' => ['/site/contact']],_/
    ];
    if (Yii::$app->user->isGuest) {

```
    //$menuItems[] = ['label' => 'Enquiry', 'url' => ['/site/acenquiry']];
    $menuItems[] = ['label' => 'Signup', 'url' => ['/site/signup']];
    $menuItems[] = ['label' => 'Login', 'url' => ['/site/login']];
} else {
    //print_r(Yii::$app->user->id);

    $menuItems[]=[
        'label'=>'Dashboard',
        'url'=>['/site/dashboard','date'=>'1','status'=>'0'],          
    ];
    $menuItems[]=['label' => 'View',
       'items' => [
           ['label' => 'Companies',
            'url' => ['/company']
           ],
           ['label' => 'Projects', 
            'url' => ['/project']
           ],
           ['label' => 'Documents', 
            'url' => ['/quotedocs']
           ],
           ['label' => 'Quotations', 
            'url' => ['/quotations']
           ],
           ['label' => 'Payments', 
            'url' => ['/grptankpayment']
           ],
       ],
       ];
    $menuItems[]=['label' => 'Configuration',
   'items' => [
       ['label' => 'Status Types',
        'url' => ['/status']
       ],
       ['label' => 'Document Types', 
        'url' => ['/doctype']
       ],
       ['label' => 'Task Types', 
        'url' => ['/tasktype']
       ],
   ],
   ];
    $menuItems[]=['label' => 'GRP',
   'items' => [
       ['label' => 'GRP Panels',
        'url' => ['/grptankpanels']
       ],
       ['label' => 'GRP Tools', 
        'url' => ['/grptanktools']
       ],
       ['label' => 'GRP Main Accessories', 
        'url' => ['/grptankaccessories']
       ],
       ['label' => 'GRP Sub Accessories', 
        'url' => ['/grptanksubaccessories']
       ],
   ],
   ];
    $menuItems[]=[
        'label'=>'Company',
        'url'=>['/company/create'],
        'linkOptions' => ['data-method' => 'post']            
    ];
    $menuItems[]=['label' => 'Project',
    'items' => [
        ['label'=>'Create Project',
        'url'=>['/project/create'], 
        ],
        ['label'=>'Project Execution',
        'url'=>['/project-execution/create'], 
        ],         
    ],
    ];
    $menuItems[] = [
        'label' => 'Quotation',
        'url' => ['/quotations/create'],
        'linkOptions' => ['data-method' => 'post']
    ];
    $menuItems[]=[
        'label'=>'Sekisui',
        'url'=>['/grptanksekisui/create'],
        'linkOptions' => ['data-method' => 'post']            
    ];

    $menuItems[] = [
        'label' => 'Docs',
        'url' => ['/quotedocs/create'],
        'linkOptions' => ['data-method' => 'post']
    ];

    $menuItems[] = [
        'label' => 'Payments',
        'url' => ['/grptankpayment/create'],
        'linkOptions' => ['data-method' => 'post']
    ];
    $menuItems[] = [
        'label' => 'My Tasks',
        'url' => ['/task'],
        'linkOptions' => ['data-method' => 'post']
    ];
    $menuItems[] = [
        'label' => 'Logout (' . Yii::$app->user->identity->username . ')',
        'url' => ['/site/logout'],
        'linkOptions' => ['data-method' => 'post']
    ];
}
/*echo Nav::widget([
    'options' => ['class' => 'navbar-nav navbar-right'],
    'items' => $menuItems,
]);*/

?>

<div class="container">
 <?php echo SideNav::widget([      
  'encodeLabels' => false,     
  'items' => $menuItems,
   'type' => SideNav::TYPE_PRIMARY
  ]); ?>
    <?= Breadcrumbs::widget([
        'links' => isset($this->params['breadcrumbs']) ? $this->params['breadcrumbs'] : [],
    ]) ?>
    <?= Alert::widget() ?>
    <?= $content ?>
</div>
```

<footer class="footer">
    <div class="container">
        <p class="pull-right">&copy; Test Company <?= date('Y') ?></p>

```
</div>
```

</footer>

<?php $this->endBody() ?>
</body>
</html>
<?php $this->endPage() ?>`
Getting outpu:
![nav error](https://cloud.githubusercontent.com/assets/10756156/18123565/111bcce6-6f8c-11e6-86d3-7b1c0b07ba5b.png)
Expected Output:
![nav1](https://cloud.githubusercontent.com/assets/10756156/18123566/111e12c6-6f8c-11e6-8a48-4b0139e8de69.png)

**Comment by ilgiz-badamshin** [comment]
Nayank9, try wrap sidebar and content in divs: 

```
<div id="container">
    <div id="sidebar-wrapper">
        <?php echo SideNav::widget([...]); ?>
    </div>
    <div id="page-content-wrapper">
        <?= Breadcrumbs::widget([...]) ?>
        <?= Alert::widget() ?>
        <?= $content ?>
    </div>
</div>
```

and set sidebar width:

```
#sidebar-wrapper {
    width: 250px;
}
```

**Comment by kartik-v** [comment]
This is resolved in new release v1.0.1. Update.
